sigle/src/pages-lib/api/feed/[username].test.ts

```diff
@@ -39,7 +39,7 @@ describe('test feed api', () => {
     expect(res.end).toBeCalled();
     const jsonObj = parser.parse(res.end.mock.calls[0][0]);
     expect(jsonObj.rss.channel).toEqual({
-      copyright: 'All rights reserved 2021, sigleapp.id.blockstack',
+      copyright: 'All rights reserved 2022, sigleapp.id.blockstack',
```

Comment:
Can you merge the latest `main` branch in your pr? This should resolve the issues.

Reply:
Sure. Will I need to revert the commit/undo the change or will it be fine to merge main into the code as it is?

Reply:
You can merge the code as it is, and resolve the conflicts is there are any :)